### ISSUE TITLE
No help when git remote can't be created

### ISSUE THREAD

**Issue by mboersma** [deis/workflow-cli#213]
The `deis` CLI used to suggest a `deis git` command to fix the situation when `deis create` collides with an existing `git` remote entry. It seems to have lost this behavior recently:
```bash
$ deis create
Creating Application... done, created famous-yardwork
Error: Error when running 'git remote add deis ssh://git@deis-builder.192.168.64.23.nip.io:2222/famous-yardwork.git'
fatal: remote deis already exists.
```

**Comment by Joshua-Anderson** [comment]
Yeah, this is some sort of regression. I'll look into it!

**Comment by mboersma** [comment]
It's not critical, but technically this is a regression so we should fix it before the Workflow v2.5.0 release.

**Comment by bacongobbler** [comment]
this is fixed in #222
